lib/ace/edit_session/bracket_match.js

```diff
@@ -37,6 +37,70 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.getBracketsCoor = function(position) {
+        if (position.column == 0) return null;
+
+
+        var charBeforeCursor,
+            matchLeft,
+            charAfterCursor,
+            matchRight;
+        
+        var length = -1,
+            depthLeft = 0,
+            depthRight = 0;
+
+
+        while (!matchLeft && !matchRight) {
+
+            length++;
+
+            charBeforeCursor = this.getLine(position.row).charAt(position.column-length);
+            charAfterCursor = this.getLine(position.row).charAt(position.column+length);
+
+            //Mby delete after
+            var temp = charBeforeCursor.match(/([\)\]\}])/);
```

Comment:
Regexp is probably not the best way to test if a single character is one of those three chars, but if you do it, I suggest using a constant compiled regexp